src/App.tsx

```diff
@@ -1,34 +1,97 @@
 /* eslint-disable max-len */
-import React from 'react';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { prepareTodos } from './helpers';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [query, setQuery] = useState('');
+  const [todoStatus, setTodoStatus] = useState<TodoStatus>(TodoStatus.ALL);
+  const [hasError, setHasError] = useState(false);
+
+  const loadTodos = useCallback(async () => {
+    setLoading(true);
+    try {
+      const loadedTodos = await getTodos();
+
+      setTodos(loadedTodos);
+      setLoading(false);
+    } catch (error) {
+      setHasError(true);
+      setLoading(false);
+    }
+  }, []);
```

Comment:
u can delete setLoading(false); from try and catch and just add it after this construction

Reply:
Good point. Done!)